### PR TITLE
Make center crop and random flip as args for unconditional image generation

### PR DESCRIPTION
**Pull request by wfng92**
Accidentally closed #2217 when syncing the fork. Given that #2042 moved ONNX-related files to a different directory, I have updated the code based on the new implementation.

Reuse the same code in `the train_text_to_image.py` script to enable/disable center crop or horizontal flip when training unconditional image generation. This comes in handy when training on character-based images that are facing a particular direction.

The training commands in the `README.md` will now include `center_crop` and `random_flip` arguments explicitly.

```
accelerate launch train_unconditional.py \
  --dataset_name="huggan/flowers-102-categories" \
  --resolution=64 --center_crop --random_flip \
  ...
```

### CODE REVIEW

File: examples/research_projects/onnxruntime/unconditional_image_generation/train_unconditional.py [hunk 1]
```diff
@@ -105,6 +97,21 @@ def parse_args():
             " resolution"
         ),
     )
+    parser.add_argument(
+        "--center_crop",
+        default=False,
+        action="store_true",
```
Comment (patrickvonplaten) [inline examples/research_projects/onnxruntime/unconditional_image_generation/train_unconditional.py:103]: Both --center_crop and --random_flip defaulted to `True` before the changes no? Could we make sure we still have the same default values to not break the scripts?

**Review by patrickvonplaten — APPROVED** [review]
Ok for me! It's slightly backwards breaking as the same script now does not automatically center_crop and random flip, but since the README is update, I think this is ok so that we can align it with text2image.

What do you think @williamberman and @patil-suraj ?

**Comment by patil-suraj** [comment]
Yeah, I agree, makes sense to align the pre-processing in the training script.

**Review by patil-suraj — APPROVED** [review]
Thanks a lot for the PR!

**Comment by wfng92** [comment]
> torchvision.transforms and the function

You are right. Created a new PR #2274 to rename `transforms` function to `transforms_images`.

**Pull request merged**